sources/core/Stride.Core/Stride.Core.csproj

```diff
@@ -16,6 +16,11 @@
     <StrideBuildTags>*</StrideBuildTags>
     <RestorePackages>true</RestorePackages>
     <ExtrasUwpMetaPackageVersion>6.2.12</ExtrasUwpMetaPackageVersion>
+    <LangVersion>12.0</LangVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net8.0|AnyCPU'">
+    <WarningLevel>7</WarningLevel>
```

Comment:
What is this for?

Reply:
it was due to an old net7 issue, seems to work without now.